API_doc/Communication.md

```diff
@@ -0,0 +1,28 @@
+# Communication lib-service
+
+All orders must contain at least **ORDER**, containing the name of the order, and **CONFIG_ID**, containing the id of the config you want to work on (except for *CONFIG_CREATE*).
```

Comment:
I think order isn't the best name, usually one uses REQUEST for clients and EVENT for the server

Reply:
I change 'order' to 'request'